server/utils/notifications/email_service.py

```
@@ -13,68 +13,49 @@
         self.smtp_password = os.getenv("SMTP_PASSWORD")
         self.from_email = os.getenv("SMTP_FROM_EMAIL")
-        self.from_name = os.getenv("SMTP_FROM_NAME")
-        self.frontend_url = os.getenv("FRONTEND_URL")
-        
-        # Validate required configuration at initialization
+        self.from_name = os.getenv("SMTP_FROM_NAME", "Aurora SRE")
+        self.frontend_url = os.getenv("FRONTEND_URL", "https://aurora-ai.net")
+
         for env_var, attr in [("SMTP_HOST", "smtp_host"), ("SMTP_USER", "smtp_user"), ("SMTP_PASSWORD", "smtp_password")]:
             if not getattr(self, attr):
                 raise ValueError(f"EmailService configuration incomplete. Missing required environment variable: {env_var}")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)